packages/website/pages/new-token.js

```diff
@@ -44,36 +45,42 @@ export default function NewToken() {
   }
 
   return (
-    <main className="bg-nsgreen">
-      <div className="mw9 center pv3 ph3 ph5-ns min-vh-100">
-        <div className="center mv4 mw6">
-          <h1 className="chicagoflf f4 fw4">New API Key</h1>
-          <form onSubmit={handleCreateToken}>
-            <div className="mv3">
-              <label htmlFor="name" className="dib mb2">
-                Name
-              </label>
-              <input
-                id="name"
-                name="name"
-                placeholder="Give this API token a name"
-                className="db ba b--black w5 pa2"
-                required
-              />
-            </div>
-            <div>
-              <Button
-                className="bg-nslime"
-                type="submit"
-                disabled={creating}
-                id="create-new-token"
-              >
-                {creating ? 'Creating...' : 'Create'}
-              </Button>
-            </div>
-          </form>
+    <div className="relative overflow-hidden">
+      <div className="layout-margins">
+        <main className="max-w-screen-lg mx-auto my-4 lg:my-16 text-w3storage-purple">
+          <div className="flex flex-col items-center justify-between mx-auto w-96 h-96 max-w-full bg-white border border-w3storage-red py-12 px-10 text-center">
+            <h3>New API Token</h3>
+            <form onSubmit={handleCreateToken}>
+              <div className="my-6">
+                <div className="text-left mb-3">
+                  <label htmlFor="name">Name</label>
+                </div>
+                <input
+                  id="name"
+                  name="name"
+                  placeholder="Give this API Token a name"
+                  className="w-full border border-black px-4 py-3 placeholder-black"
+                  required
+                />
+              </div>
+              <div>
+                <Button
+                  type="submit"
+                  disabled={creating}
+                  id="create-new-token"
+                >
+                  {creating ? 'Creating...' : 'Create'}
+                </Button>
+              </div>
+            </form>
+          </div>
+        </main>
+        <div className="absolute top-48 left-0 w-full pointer-events-none" style={{ minWidth: '1536px' }}>
+          <div className="w-min ml-auto">
+            <VerticalLines />
```

Comment:
This illustration is cropping because of the overflow-hidden in the parent, but not too important right now for the first delivery 
![image](https://user-images.githubusercontent.com/24696635/126779877-76e901e1-1fb7-46b9-9ee4-4ec1f6bc1a84.png)